<launch>
  <param name="halfWidth"     type="double"   value="0.25" />
  <param name="trackShape"  type="str"    value="oval_iri"/>
  <node pkg="plotter" type="fisheyecam_plot.py" name="cam_pose_plotter" output="screen">
  <param name="track_visualization"    type="bool"      value="true" />

  <param name="pose_visualization"    type="bool"      value="true" />
  <param name="record_on"     		 type="bool"      value="false" />
  <param name="loop_rate"     		 type="int"      value="1000" />

  </node>

</launch>
